resources/templates/base.html

```diff
@@ -172,14 +172,17 @@
 
            if (balance && balance.ETH && eth_balance_display) {
              eth_balance_display.textContent = balance && balance.ETH.formatted;
+             eth_balance_display.textContent += ` (${balance.ETH.as_fiat} $)`;
```

Comment:
If there is no CMC api key, this part should not be displayed! Right now it would add `"  $"` to `textContent`, if the CMC api call would not return a value.
Same for the other `.as_fiat` formattings below and in `launch.html`!